### PR TITLE
[docs] Added remarks about wildcard usage. Added no_proxy variable in helm_lib.

### PR DESCRIPTION
**Pull request by duckhawk**
## Description
Added remarks about wildcard usage in [the Getting Started doc](https://deckhouse.io/en/gs/). Also, added environment variables no_proxy, http_proxy, https_proxy for better compatibility ([here](https://about.gitlab.com/blog/2021/01/27/we-need-to-talk-no-proxy/) are some details).

## Why do we need it, and what problem does it solve?
There was no information about wildcard in Getting Started doc. Variables no_proxy, http_proxy and https_proxy were added for better [compatibility](https://about.gitlab.com/blog/2021/01/27/we-need-to-talk-no-proxy/).

## What is the expected result?
There is information about wildcard in Getting Started doc. Also, in case of wget and curl utility usage inside pods, proxy (and proxy ignore) will work for multiple modules.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
```changes
section: docs
type: chore
summary: Added information about using wildcards in the Getting Started.
impact_level: low
---
section: cloud-provider-yandex
type: fix
summary: In case of wget and curl utility usage inside pods, proxy (and proxy ignore) will work.
impact: The `cloud-provider-yandex` module will be restarted if a proxy is enabled in the cluster.
---
section: linstor
type: fix
summary: In case of wget and curl utility usage inside pods, proxy (and proxy ignore) will work.
impact: The `linstor` module will be restarted if a proxy is enabled in the cluster.
---
section: snapshot-controller
type: fix
summary: In case of wget and curl utility usage inside pods, proxy (and proxy ignore) will work.
impact: The `snapshot-controller` module will be restarted if a proxy is enabled in the cluster.
```

### CODE REVIEW

File: docs/site/_includes/getting_started/global/step_cluster_setup.html [hunk 1]
```diff
@@ -153,7 +153,7 @@
         name="modulesNoProxyAddressList" placeholder=""
         autocomplete="off" />
       <span class="info">
-         Specify a list of IP addresses, networks, and domain names that can be accessed directly without using a proxy server. Leave it blank if there are no such exceptions.
+         Specify a list of IP addresses, networks, and domain names that can be accessed directly without using a proxy server. For wildcard, you can use domain with dot in the beginning, like ".example.com". Leave it blank if there are no such exceptions.
```
Comment (apolovov) [inline docs/site/_includes/getting_started/global/step_cluster_setup.html:156]: There also is https://deckhouse.io/en/gs/bm-private/step3.html

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'deploy/web/test'...

File: docs/site/_includes/getting_started/global/step_cluster_setup_ru.html [hunk 2]
```diff
@@ -145,15 +145,15 @@
...
+        Список IP-адресов и доменных имен для которых проксирование не применяется, через запятую. Для всех доменов третьего уровня (wildcard), используйте домен второго, добавив точку в начале (например — <code>127.0.0.1, 192.168.0.0/24, example.com, ".example.com"</code>)
       </label>
       <input
         class="textfield"
         type="text" id="modulesNoProxyAddressList"
         name="modulesNoProxyAddressList" placeholder=""
         autocomplete="off" />
       <span class="info">
-         Укажите список IP-адресов, сетей и доменных имен, к которым есть доступ напрямую, без использования proxy-сервера. Оставьте пустым, если таких исключений нет.
+         Укажите список IP-адресов, сетей и доменных имен, к которым есть доступ напрямую, без использования proxy-сервера. Для всех доменов третьего уровня (wildcard), используйте домен второго, добавив точку в начале, например ".example.com". Оставьте пустым, если таких исключений нет.
```
Comment (apolovov) [inline docs/site/_includes/getting_started/global/step_cluster_setup_ru.html:156]:
```suggestion
         Укажите список IP-адресов, сетей и доменных имен, к которым есть доступ напрямую, без использования proxy-сервера. Для настройки wildcard-доменов используйте написение вида ".example.com". Оставьте пустым, если таких исключений нет.
```

File: global-hooks/openapi/config-values.yaml [hunk 3]
```diff
@@ -220,7 +220,7 @@ properties:
               Proxy URL for HTTPS requests.
           noProxy:
             description: |
-              List of no proxy IP and domain entries.
+              List of no proxy IP and domain entries. For wildcard, you can use domain with dot in the beginning, like ".example.com".
```
Comment (kirkonru) [inline global-hooks/openapi/config-values.yaml:223]:
```suggestion
              List of no-proxy IP and domain entries. You can use a domain with a dot prefix as a wildcard, e.g., ".example.com".
```

File: global-hooks/openapi/doc-ru-config-values.yaml [hunk 4]
```diff
@@ -116,4 +116,4 @@ properties:
               Proxy URL для HTTPS-запросов.
           noProxy:
             description: |
-              Список IP и доменных имен для которых проксирование не применяется.
+              Список IP и доменных имен для которых проксирование не применяется. Для всех доменов третьего уровня (wildcard), используйте домен второго, добавив точку в начале, например ".example.com".
```
Comment (kirkonru) [inline global-hooks/openapi/doc-ru-config-values.yaml:119]:
```suggestion
              Список IP и доменных имен, для которых проксирование не применяется. Для всех доменов третьего уровня (wildcard), используйте домен второго, добавив точку в начале, например ".example.com".
```
Perhaps it makes sense to rewrite it as _`Для настройки wildcard-доменов используйте написание вида ".example.com"`_ (see apolovov's suggestion above).

File: global-hooks/openapi/config-values.yaml (hunk 3 above)
Comment (kirkonru) [reply global-hooks/openapi/config-values.yaml:223]: Well, I don't really like the way it sounds. How'bout `For wildcard-domains, use a domain name with a dot prefix, e.g., ".example.com"`?

**Review by kirkonru — APPROVED** [review]

**Review by apolovov — APPROVED** [review]

**Pull request merged**